src/Kuzzle.js

```diff
@@ -460,20 +469,23 @@ Discarded request: ${JSON.stringify(request)}`));
       throw new Error('You must provide a valid accessor.');
     }
 
-    if (this[accessor]) {
+    if (this.__proxy__ ? this.__proxy__.hasProp(accessor) : this[accessor]) {
```

Comment:
Why checking it the object is proxified ? Since we return the proxified object in the constructor it should always be the case no?

Reply:
Because this method is used in the constructor when the object is not yet proxified